dev-util/clion/clion-2024.3.2.ebuild

```
@@ -0,0 +1,107 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop wrapper
+
+DESCRIPTION="A complete toolset for C and C++ development"
+HOMEPAGE="https://www.jetbrains.com/clion/"
+SRC_URI="https://download.jetbrains.com/cpp/CLion-${PV}.tar.gz"
+
+LICENSE="|| ( IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )
+	Apache-1.1 Apache-2.0 BSD BSD-2 CC0-1.0 CDDL-1.1 CPL-0.5 CPL-1.0
+	EPL-1.0 EPL-2.0 GPL-2 GPL-2-with-classpath-exception GPL-3 ISC JDOM
+	LGPL-2.1+ LGPL-3 MIT MPL-1.0 MPL-1.1 OFL-1.1 public-domain PSF-2
+	UoI-NCSA ZLIB"
+SLOT="0/2024"
+KEYWORDS="~amd64"
+RESTRICT="bindist mirror splitdebug"
+
+BDEPEND="dev-util/patchelf"
+
+RDEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0:2
+	dev-debug/gdb
+	dev-libs/expat
+	dev-libs/glib:2
+	dev-util/lttng-ust:0/2.13
```

```suggestion
	dev-util/lttng-ust:0/2.12
```
As far as I can see, `/opt/clion/plugins/clion-radler/DotFiles/linux-x64/dotnet/shared/Microsoft.NETCore.App/8.0.11/libcoreclrtraceptprovider.so` needs `liblttng-ust.so.0`, which is in `dev-util/lttng-ust-2.12.10`